dev-util/sourcetrail/patches/sourcetrail-2019.4.102.patchset

```diff
@@ -0,0 +1,21 @@
+From 2a996ed9a07c069b5920801c28edfae5cc7106e8 Mon Dec 16 15:38:31 2019 +0100
+Author: Ruixuan Tu <turx2003@gmail.com>
+Date: Tue Jan 14 01:00:00 2020 +0800
+Subject: fix dependency of librt
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index aaf04cf..911d59f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -230,7 +230,7 @@ target_include_directories(${LIB_UTILITY_PROJECT_NAME} SYSTEM
+ 
+ target_link_libraries(${LIB_UTILITY_PROJECT_NAME} ${Boost_LIBRARIES} Qt5::Widgets Qt5::Network)
+ 
+-if (UNIX AND NOT APPLE)
++if (UNIX AND NOT APPLE AND NOT HAIKU)
+        find_package(Threads REQUIRED)
+        target_link_libraries(${LIB_UTILITY_PROJECT_NAME} ${CMAKE_DL_LIBS} rt ${CMAKE_THREAD_LIBS_INIT})
+ endif()
+--
+2019.4.102
```

Comment:
This doesn't look right to me, did you edit it?

Reply:
I did not edit it.

Reply:
From your log:
```
Skipping unpack of 2019.4.102.tar.gz
HEAD is now at 4ae2f02 import
Applying patchset "/boot/home/haikuports/dev-util/sourcetrail/patches/
```
Try to clean your build and then try again, looks it's trying to patch an already patched file?
Also, add the alias "hp" to your profile (saves you adding the options to run haikuporter), how to do that is explained in the "haikuports.conf" file

Reply:
I have cleaned the build.
When it is the first time, the error message is "fatal: empty ident name (for <>) not allowed."
When it is not the first time, the error message is "fatal: previous rebase directory .git/rebase-apply still exists but mbox given."